### PR TITLE
Example APK for recovery testing

### PR DESCRIPTION
**Pull request by nautomm**
Features:
- Example APK for recovery testing
- APK has been Integrated with CameraModule

### CODE REVIEW

File: app-mp4-recovery/src/main/java/com/nauto/camera/mp4/app/MainActivity.java [hunk 1]
```diff
@@ -1,23 +1,192 @@
...
+                if (Mp4RecoveryJNI.isCorrect(inputFile))
+                    stopOperationProgress("Input file is valid.");
+                else
+                    stopOperationProgress("Input file is corrupted.");
+            }
+        });
     }
 
+    private void recoverInputFile() {
+        startOpeationProgress("Beginning file recovery...");
```
Comment (ab2005) [inline app-mp4-recovery/src/main/java/com/nauto/camera/mp4/app/MainActivity.java]: No need for the progress if the operation takes less then 1 sec.

File: app-mp4-recovery/src/main/java/com/nauto/camera/mp4/app/MainActivity.java [hunk 2]
```diff
@@ -1,23 +1,192 @@
...
+
+        new Handler(Looper.getMainLooper()).post(new Runnable() {
```
Comment (ab2005) [inline app-mp4-recovery/src/main/java/com/nauto/camera/mp4/app/MainActivity.java]: The Handler needs to be a class member.

File: app-mp4-recovery/src/main/java/com/nauto/camera/mp4/app/MainActivity.java (hunk 1 above)
Comment (nautomm) [reply app-mp4-recovery/src/main/java/com/nauto/camera/mp4/app/MainActivity.java]: For larger files like third example recovery takes about 3-4 sec.

File: app-mp4-recovery/src/main/java/com/nauto/camera/mp4/app/MainActivity.java (hunk 2 above)
Comment (nautomm) [reply app-mp4-recovery/src/main/java/com/nauto/camera/mp4/app/MainActivity.java]: Right thing! I'll repair it later

**Pull request merged**